diff --git a/node_modules/onnxruntime-react-native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java b/node_modules/onnxruntime-react-native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java
--- a/node_modules/onnxruntime-react-native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java
+++ b/node_modules/onnxruntime-react-native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java
@@ -47,8 +47,8 @@
     try {
       System.loadLibrary("onnxruntimejsi");
       JavaScriptContextHolder jsContext = getReactApplicationContext().getJavaScriptContextHolder();
-      CallInvokerHolderImpl jsCallInvokerHolder =
-        (CallInvokerHolderImpl) getReactApplicationContext().getCatalystInstance().getJSCallInvokerHolder();
+      CallInvokerHolderImpl jsCallInvokerHolder =
+        (CallInvokerHolderImpl) getReactApplicationContext().getJSCallInvokerHolder();
       nativeInstall(jsContext.get(), jsCallInvokerHolder);
       return true;
     } catch (Exception e) {
